# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/worklet/WorkletGlobalScope.cpp
# Commit: a8e623b228ae
# Full Hash: a8e623b228aeb04c741bebeeaea4f2524b2ddcc7
# Author: Tom Ritter <tom@mozilla.com>
# Date: 2022-11-29 21:47:49
# Regressor Bug: 1778510
# File Overlap Count: 1
# Description:
#   Bug 1778510: Add RTPCallerType to GlobalObject r=asuth
#   
#   This centralizes the logic in one place.
#   
#   In order to do this, we will need to check the principal
# ==============================================================================

diff -r 32fdf6f014f7 -r a8e623b228ae dom/worklet/WorkletGlobalScope.cpp
--- a/dom/worklet/WorkletGlobalScope.cpp	Tue Nov 29 13:34:14 2022 +0000
+++ b/dom/worklet/WorkletGlobalScope.cpp	Tue Nov 29 13:34:15 2022 +0000
@@ -76,6 +76,10 @@
   return mImpl->ShouldResistFingerprinting();
 }
 
+bool WorkletGlobalScope::IsSystemPrincipal() const {
+  return mImpl->IsSystemPrincipal();
+}
+
 void WorkletGlobalScope::Dump(const Optional<nsAString>& aString) const {
   WorkletThread::AssertIsOnWorkletThread();
 